### DISCUSSION TITLE
I see fewer items in the feed

### DISCUSSION THREAD

**Discussion by avdosev** [community/community#13319] [Other Feature Feedback, Questions, & Ideas]
In the old feed, I can see 5 items and part of the 6 item

![old feed](https://user-images.githubusercontent.com/39769345/159669434-27918e64-154d-43ea-9c91-dff9dc9f3acf.png)

But in the new feed, I can see 4 items and part of the 5 item

![new feed](https://user-images.githubusercontent.com/39769345/159669293-d13ab287-80ff-43ef-afb1-f2b0884f83e3.png)

**Comment by sntwandy** [comment]
It is a very specific factor, the new design provides more relevant and consistent information which could be a little more useful, I do not think it matters much now the margin of whether one more card is shown or not, but it is interesting to study and see this action.

**Reply by avdosev** [reply]
In my case, the screen had almost the same information, but there was less of it for the new design. (I agree that newest feed is more informative.)

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
